.github/workflows/appimage.yml

```diff
@@ -52,37 +60,29 @@ jobs:
       run: |
         sudo -u testbot bash -lc 'make setup'
```

Comment:
line 57 runs `make clean`. Should this be `make clean-all`? It shouldn't matter as this is a new canon but `make clean-all` is more thorough